tests/sensu-client-windows.pp

```diff
@@ -0,0 +1,9 @@
+# Note: Check http://repositories.sensuapp.org/msi/ for the latest version.
+class { '::sensu':
+  version           => '0.29.0-11',
+  rabbitmq_password => 'correct-horse-battery-staple',
+  rabbitmq_host     => '192.168.56.10',
+  rabbitmq_vhost    => '/sensu',
+  subscriptions     => 'all',
+  client_address    => $facts['networking']['ip'],
```

Comment:
In your vagrant, this should use the 192. IP and not the 10.

I check in http://localhost:3000/#/clients